src/main/java/no/ntnu/mycbr/rest/controller/RetrievalController.java

```diff
@@ -91,6 +91,38 @@ public Query getSimilarCasesByID(
             @ApiResponse(code = 500, message = "Failure")
     })
     public HashMap<String, HashMap<String,Double>> getSimilarInstancesByIDWithinIDs(
+            @PathVariable(value="conceptID") String conceptID,
+            @PathVariable(value="casebaseID") String casebaseIDs,
```

Comment:
casebaseIDs should be casebaseID as there is only one.

Reply:
Agree.
I will submit a new commit when we resolve the parameter naming.